src/event_handler.rs

```diff
@@ -16,6 +16,7 @@ use crate::{
 };
 
 use self::ready::ready_handler;
+pub static mut interaction_count: i32 = 0;
```

Comment:
Rust pretty much recommends you do NOT do something like this, additionally it is not thread safe. You would want to wrap this in an [ARC](https://doc.rust-lang.org/std/sync/struct.Arc.html) much like Mainline Luro does for it to be thread safe. HOWEVER, this does not fix the fact that static variables should not be modified like this, so the real fix is below.

Extend the Luro struct (The one that contains `application_id`, `http` and such in `main.rs`) and add this variable. Wrap it in [RwLock](https://doc.rust-lang.org/std/sync/struct.RwLock.html) much like some of the other variables, then you can modify it from within Luro's context. The other commands should help you get a grasp of how to achieve that, such as the hecks command.

Reply:
I actually edited the luro struct at first but couldn't get it to work. Since it was the first command I made I was hoping to spend more time debugging my twilight implentation instead of my own code. I like your standards and the link is helpful. I'll bump the priority of fixing this.